apps/drec-api/src/pods/organization/dto/organization.dto.ts

```diff
@@ -13,36 +14,58 @@ export class OrganizationDTO
   extends PublicOrganizationInfoDTO
   implements IFullOrganization
 {
-  @ApiProperty({ type: String })
-  @IsEthereumAddress()
-  @Expose()
-  blockchainAccountAddress: string;
-
   @ApiProperty({ type: String })
   @IsString()
+  @Expose()
   signatoryFullName: string;
 
   @ApiProperty({ type: String })
   @IsString()
+  @Expose()
   signatoryAddress: string;
 
   @ApiProperty({ type: String })
   @IsString()
+  @Expose()
   signatoryZipCode: string;
 
   @ApiProperty({ type: String })
   @IsString()
+  @Expose()
   signatoryCity: string;
 
   @ApiProperty({ type: String })
   @IsISO31661Alpha2()
+  @Expose()
   signatoryCountry: string;
 
   @ApiProperty({ type: String })
   @IsEmail()
+  @Expose()
   signatoryEmail: string;
 
   @ApiProperty({ type: String })
   @IsString()
+  @Expose()
   signatoryPhoneNumber: string;
+
+  @ApiPropertyOptional({ type: [String] })
+  @IsArray()
+  @Expose()
+  documentIds?: string[];
+
+  @ApiPropertyOptional({ type: [String] })
+  @IsArray()
+  @Expose()
+  signatoryDocumentIds?: string[];
```

Comment:
What are these documents used for? Are they related to the blockchain address?

Reply:
I`m not sure. They might be documents related to the lead user. But can be used to replace the documentIds to prove company validity.